source/src/sidescroller/PClass.java

```diff
@@ -73,12 +72,12 @@ public void background(int g) {
 
 	/**
 	 * Sets the background image of the applet.
-	 * @param img The image to use as PGraphics
+	 * @param icon_eyeActive The image to use as PImage
```

Comment:
Why "Sets the background image of the applet."? It simply draws an image at (x, y). Also why `icon_eyeActive`?

Reply:
I didn't write the Javadoc for that function. Clearly, someone wrote the the Javadoc, thinking it was a different method.

Not sure why it's 'icon_eyeActive' -- perhaps it was on my clipboard at some time. It can be changed to 'img'.